web-admin/tests/setup/teardown.ts

```diff
@@ -1,20 +1,29 @@
 import { expect } from "@playwright/test";
 import { execAsync } from "@rilldata/web-common/tests/utils/spawn";
 import fs from "fs";
-import { test as teardown } from "./base";
+import { join } from "node:path";
+import { test as teardown, TestTempDirectory } from "./base";
 import { RILL_DEVTOOL_BACKGROUND_PROCESS_PID_FILE } from "./constants";
 
 teardown.describe("global teardown", () => {
   teardown("should clean up the test organization", async ({ cli: _ }) => {
     await execAsync("rill org delete e2e --interactive=false");
+    await execAsync(
+      // We need to set the home to get the correct creds
+      `HOME=${join(TestTempDirectory, "deploy_home")} rill org delete e2e-viewer --interactive=false`,
+    );
 
     // Wait for the organization to be deleted
     // This includes deleting the org from Orb and Stripe, which we'd like to do to keep those environments clean.
     await expect
-      .poll(async () => await isOrgDeleted("e2e"), {
-        intervals: [1_000],
-        timeout: 15_000,
-      })
+      .poll(
+        async () =>
+          (await isOrgDeleted("e2e")) || (await isOrgDeleted("e2e-viewer")),
```

Comment:
This `||` confuses me. Reading the test, I'm not sure which org is expected to be deleted. Is it meant to be an `&&`?